src/GraphWebhooks-Core/Controllers/AccountController.cs

```diff
@@ -3,49 +3,27 @@
 *  See LICENSE in the source repository root for complete license information. 
 */
 
-using System.Threading.Tasks;
```

Comment:
Do you still need the AccountController?

Reply:
Yes, for the callback and redirect actions (**GrantPermissions**, **RequestPermissions**) from and to the _adminconsent_ endpoint respectively.